WordPress/src/main/java/org/wordpress/android/ui/sitecreation/SiteCreationBaseFormFragment.java

```diff
@@ -47,14 +47,15 @@ protected ViewGroup createMainView(LayoutInflater inflater, ViewGroup container,
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         ViewGroup rootView = createMainView(inflater, container, savedInstanceState);
-        setupContent(rootView);
         return rootView;
     }
 
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
+        setupContent((ViewGroup) getView().getRootView());
```

Comment:
Would it be better to use the `view` argument here instead? AS is complaining about a potential NPE here.

```suggestion
        setupContent((ViewGroup) view);
```

Reply:
I think passing `view` would also probably work, but I'd rather not change this just to be safe. (+ to save some time testing the change)

Reply:
I actually tested this before I made the suggestion, but it was not an important point. Thank you for updating the rest of the code and making AS happy.